src/app/core/access-checker/cache/cache-entry.model.js

```diff
@@ -20,7 +20,7 @@ const CacheEntrySchema = new mongoose.Schema({
 	// The actual ts this entry was entered into the cache
 	ts: {
 		type: Date,
-		default: Date.now,
+		default: () => Date.now(),
```

Comment:
@brianghig I made that change based on the recommendations in migration guide[1].  In the case of `Date.now`, it would have been safe to leave as is, since it doesn't take any parameters.  However, we also have a default set of `uuid.v4` which does take parameters and could be affected.  Decided to be consistent and just update all instances.

[1] https://mongoosejs.com/docs/migrating_to_6.html#document-parameter-to-default-functions.